server/services/FiltersService/index.js

```diff
@@ -66,149 +76,41 @@ class FiltersService {
-            study: { $in: this.filters.sites },
-          },
-        },
-      ]
-    }
-    if (isIncOrExcOnly || isCritOnly) {
-      const variable = chrCritFilters.length
-        ? 'chrcrit_part'
-        : 'included_excluded'
-      const values = chrCritFilters.length
-        ? chrCritFilters
-        : includedExcludedFilters
-
-      $facet.formInclusion = [
-        {
-          $match: {
-            assessment: INCLUSION_EXCLUSION_CRITERIA_FORM,
-            study: {
-              $in: this.filters.sites,
-            },
-            dayData: {
-              $elemMatch: { [variable]: { $in: values } },
-            },
-          },
-        },
-      ]
-    }
-    if (sexAtBirthFilters.length) {
-      $facet.sexAtBirth = [
-        {
-          $match: {
-            assessment: SOCIODEMOGRAPHICS_FORM,
-            dayData: {
-              $elemMatch: {
-                chrdemo_sexassigned: { $in: sexAtBirthFilters },
-              },
-            },
-          },
-        },
-      ]
-    }
-    return Object.keys($facet).length > 1
-      ? [
-          { $facet },
-          {
-            $project: {
-              participants: {
-                $setIntersection: [
-                  '$formInclusion.participant',
-                  '$sexAtBirth.participant',
-                ],
-              },
-            },
-          },
-        ]
-      : $facet
-  }
+  get filterQueries() {
+    const includedExcludedQuery = (includedValues) => ({[ `${DAY_DATA_KEY}.${INCLUSION_EXCLUSION_KEY}`]: { $in: includedValues } })
+    const chrChritQuery = (includedValues) => ({[ `${DAY_DATA_KEY}.${CHRCRIT_KEY}`]: { $in: includedValues } })
+
+    const sexAtBirthQuery = (includedValues) => ({ [`${DAY_DATA_KEY}.${SEX_AT_BIRTH_DOCUMENT_KEY}`]: {$in: includedValues } })
 
-  _buildFacetForFilters = ({
-    isSexAtBirthFilterActive,
-    isInclusionCriteriaFilterActive,
-    sites,
-  }) => {
-    const facetForFilters = {}
-    if (isSexAtBirthFilterActive) {
-      facetForFilters.socioDemographics = [
-        {
-          $match: {
-            assessment: SOCIODEMOGRAPHICS_FORM,
-            study: { $in: sites },
-          },
-        },
-        {
-          $project: {
-            ...INDIVIDUAL_FILTERS_MONGO_PROJECTION,
-          },
-        },
-        {
-          $addFields: {
-            filter: SOCIODEMOGRAPHICS_FORM,
-          },
-        },
-      ]
+    const filterQueries = []
+    const filterNames = new Set(Object.keys(this.filters).filter(key => (this.filters[key].map(f => f.value).indexOf(TRUE_STRING) > -1)))
+
+    if (filterNames.has(INCLUSION_EXCLUSION_KEY) && filterNames.has(CHRCRIT_KEY)) {
+      filterQueries.push({
+        assessment: INCLUSION_EXCLUSION_CRITERIA_FORM,
+        ...includedExcludedQuery(filterToMongoValues(this.filters[INCLUSION_EXCLUSION_KEY])),
+        ...chrChritQuery(filterToMongoValues(this.filters[CHRCRIT_KEY]))
+      })
+    } else if (filterNames.has(INCLUSION_EXCLUSION_KEY)) {
```

Comment:
just checking that these should all be `else if`s so only one is applied vs `if`s where multiple could be applied.